src/vue/forms/InvestForm.vue

```diff
@@ -0,0 +1,564 @@
+<template>
+  <div class="invest-form">
+    <template v-if="isLoaded && quoteAssetBalances.length">
+      <form
+        novalidate
+        class="app__form"
+        @submit.prevent="isFormValid() && showConfirmation()"
+      >
+        <div class="app__form-row">
+          <div class="app__form-field">
+            <!--
+              :key is a hack to ensure that the component will be updated
+              after property change
+            -->
+            <select-field
+              :key="form.asset"
+              v-model="form.asset"
+              :values="quoteAssetListValues"
+              :label="'invest-form.asset-lbl' | globalize"
+              id="invest-asset"
+              @blur="touchField('form.asset')"
+              :disabled="formMixin.isDisabled || !canUpdateOffer"
+            />
+
+            <vue-markdown
+              class="app__form-field-description invest-form__amount-hint"
+              :source="'invest-form.available-amount-hint' | globalize({
+                amount: availableAmount
+              })"
+              :html="false"
+            />
+          </div>
+        </div>
+
+        <vue-markdown
```

Comment:
Idk how it looks, but this message seems to be redundant. If the offer exist, i think it's enough to just place it's amount to the amount field and change button text from `Invest` to `Update`

@max-nes

Reply:
In that way, we should update amount after every asset change, so user's entered data will be lost. We should store user's input irrespective of selected asset.